app/models/stop_searcher.rb

```diff
@@ -2,18 +2,18 @@ class StopSearcher
   include ActiveModel::SerializerSupport
   DEFAULT_PER_PAGE = 20
 
+  attr_reader :per_page, :page
+
   def initialize(params)
     @params = params
+    @per_page = params.fetch(:per_page) { DEFAULT_PER_PAGE }.to_i
+    @page = params.fetch(:page) { 1 }.to_i
   end
```

Comment:
I don't have any reason other than I forget about the two argument form from time to time. Avdi has some reasons on page 128 of Confident Ruby in the section Two-argument #fetch.